### ISSUE TITLE
Flaky test caused by invalid alias name error

### ISSUE THREAD

**Issue by villasv** [gupy-io/searchops#184]
```
● Index Migrations › Updating dynamic index settings › given the index was created › when the manager performs a migration › then the index should exist

    ResponseError: invalid_alias_name_exception
``` 

https://github.com/gupy-io/searchops/runs/1936859362#step:6:105

**Comment by villasv** [comment]
It turns out the problem is not really the name, but the fact that it already exists:

```
"body": {
    "error": {
      "root_cause": [
        {
          "type": "invalid_alias_name_exception",
          "reason": "Invalid alias name [account_1613754622454], an index exists with the same name as the alias",
          "index_uuid": "Qc6lTq3iSsGkF6kf4yiFsg",
          "index": "account_1613754622454"
        }
      ],
      "type": "invalid_alias_name_exception",
      "reason": "Invalid alias name [account_1613754622454], an index exists with the same name as the alias",
      "index_uuid": "Qc6lTq3iSsGkF6kf4yiFsg",
      "index": "account_1613754622454"
    },
    "status": 400
  },
```

**Comment by villasv** [comment]
Maybe caused by tests with the same label.

**Comment by villasv** [comment]
Supposedly closed in 5cb6522c464a688f91707c041285aa305c0d556d
